examples/lighting-app/bouffalolab/bl602/src/AppTask.cpp

```diff
@@ -138,7 +138,9 @@ void AppTask::AppTaskMain(void * pvParameter)
     log_info("App Task entered\r\n");
 
     sWiFiNetworkCommissioningInstance.Init();
-    chip::Server::GetInstance().Init();
+    chip::CommonCaseDeviceServerInitParams initParams;
+    initParams.InitializeStaticResourcesBeforeServerInit();
+    chip::Server::GetInstance().Init(initParams);
```

Comment:
Does Init return CHIP_ERROR? Could you handle it (at least log on error)?

Reply:
> @jczhang777 the summary now says that pbuf size was updated to match matter. Where is that update? I wanted to make sure that if 2 sizes need to be in sync, there is a comment explaining this.
> 
> is this the later SDK repo update or something else?

When lwIP PBUf alloc RAW, the length of PBUF_LINK_ENCAPSULATION_HLEN must be reserved 128 bytes that is sufficient.
I'm sorry for the trouble you by not describing clearly.

Reply:
I have solved the problem as you said. Thank you very much for your careful review. I'll pay attention to that next time.